ably/transport/defaults.py

```diff
@@ -47,7 +47,8 @@ def get_port(options):
 
     @staticmethod
     def get_fallback_rest_hosts(options):
-        if options.rest_host:
+        # RTN17b
```

Comment:
Not sure why this is part of this PR, however in spite of this:

* There is no test for this
* The spec code should go with the test, not the library code unless there is a good reason (perhaps to explain some complex logic)
* The spec item states: `The fallback behavior described below only applies when the default realtime.ably.io endpoint is being used and has not been overriden (see RTC1d and RTC1e), ClientOptions#fallbackHostsUseDefault is true, or an array of ClientOptions#fallbackHosts is provided.`.  So `fallBackHostsUseDefault` is not implemented, neither is the optional array of `fallbackHosts`.  So this commit is really not great as it partly implements a spec and has no tests to confirm it works